html/swipe.html

```diff
@@ -0,0 +1,137 @@
+                <img src="../assets/slack.png" alt="" class="item item4">
+                <img src="../assets/shopify.png" alt="" class="item item5">
+                <img src="../assets/google.png" alt="" class="item item6">
+                <img src="../assets/lyft.png" alt="" class="item item7">
+                <img src="../assets/zoom.png" alt="" class="item item8">
+            </div>
+        </div>
+    </div>
+    </main>
+    <!-- UNIFIED PLATFORM = UP -->
+    <div class="background-2">
+        <div class="bgf-section2">
+          <div class="section-up">
+            <div class="container-up">
+                <p class="subtitle">
+                unified platform
+                </p>
+                <h2 class="h2-section2">
+                All the payment products you'll
+                    <br>
+                ever need
+                </h2>
+            <div class="columns-p">
+                <div class="col-left2">
+                    <p>We've got everything you'll need and more to buil websites and apps that handle payments, payouts, and much more. Swipe's products power payments for 
+                        <a href="#">all kinds of retailer, all kinds of businesses, software platforms & marketplaces, </a>and everything in between.</p>
+                </div>        
+                <div class="col-right2">
+                    <p>We also help companies 
+                        <a href="#">tackle fraud, issue invoices, sent virtual and physical cards, manage financing, wrangle business spend, </a>
+                    and much more.
+                    </p>
+                </div>
+            </div>
+                <button class="btn-start2">
+                        Start now
+                    <img class="start2-flecha2" src="../assets/arrow-right-light.svg" alt="flecha-derecha">
+                </button>
+                <!-- IMGS SECTION-2 -->
+                <div class="imgs-section2">
+                    <div class="container-imgs2">
+                        <div class="grid-imgs2">
+                            <img class="card" src="../assets/credit-card.png" alt="">
+                            <img class="slack" src="../assets/slack-invoice.png" alt="">
+                            <img class="phone2" src="../assets/phone.png" alt="">
+                            <img class="dash" src="../assets/dash.png" alt="">
+                        </div>
+                    </div>
+                </div>
+            </div>
+          </div>
+        </div>
+    </div>
+</body>
+<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js" integrity="sha384-I7E8VVD/ismYTF4hNIPjVp/Zjvgyol6VFvRkX/vR+Vc4jQkC+hVqc2pM8ODewa9r" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.min.js" integrity="sha384-fbbOQedDUMZZ5KreZpsbe1LCZPVmfTnH7ois6mU1QK+m14rQ1l2bGBq41eYeM/fS" crossorigin="anonymous"></script>
```

Comment:
Una pregunta Will... de que son estos Scripts?

Reply:
Dani son unas librerias de boostrap que no estoy usando en realidad. Tan solo afecta un botón. El primer botón de la seccion 1  Sing-in. Las dejé porque cuando pregunté ya había avanzado pero en realidad no estoy llamando ninguna función de esa librería.